test/functional/pdo_sqlsrv/skipif.inc

```diff
@@ -1,4 +1,4 @@
 <?php
-if (!extension_loaded("pdo") || !extension_loaded('pdo_sqlsrv'))
-    die("PDO driver cannot be loaded; skipping test.\n");
-?>
+if (!extension_loaded("pdo_sqlsrv")) {
+    die("skip Extension not loaded");
+}
```

Comment:
why get rid of the closing `?>`?

Reply:
Because the closing tag of a PHP block at the end of a file is actually optional